modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/upgrade/v5_0_0/CommerceRegionUpgradeProcess.java

```diff
@@ -0,0 +1,90 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.internal.upgrade.v5_0_0;
+
+import com.liferay.commerce.internal.upgrade.base.BaseCommerceServiceUpgradeProcess;
+import com.liferay.commerce.model.CommerceRegion;
+import com.liferay.commerce.model.CommerceRegionLocalization;
+import com.liferay.commerce.model.impl.CommerceRegionImpl;
+import com.liferay.commerce.service.CommerceRegionLocalService;
+import com.liferay.commerce.service.persistence.CommerceRegionLocalizationPersistence;
+import com.liferay.counter.kernel.service.CounterLocalService;
+import com.liferay.portal.kernel.dao.orm.QueryUtil;
+import com.liferay.portal.kernel.language.Language;
+import com.liferay.portal.kernel.language.LanguageUtil;
+import com.liferay.portal.kernel.util.LocaleUtil;
+
+import java.util.List;
+
+/**
+ * @author Luca Pellizzon
+ */
+public class CommerceRegionUpgradeProcess
+	extends BaseCommerceServiceUpgradeProcess {
+
+	public CommerceRegionUpgradeProcess(
+		CommerceRegionLocalService commerceRegionLocalService,
+		CommerceRegionLocalizationPersistence
+			commerceRegionLocalizationPersistence,
+		CounterLocalService counterLocalService) {
+
+		_commerceRegionLocalService = commerceRegionLocalService;
+		_commerceRegionLocalizationPersistence =
+			commerceRegionLocalizationPersistence;
+		_counterLocalService = counterLocalService;
+	}
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		Language language = LanguageUtil.getLanguage();
+
+		String languageId = language.getLanguageId(LocaleUtil.US);
+
+		addColumn(
+			CommerceRegionImpl.class, CommerceRegionImpl.TABLE_NAME,
+			"defaultLanguageId", "VARCHAR(75) default " + languageId);
```

Comment:
Hey @luca-pellizzon sorry was looking at this again, I think you are going to have to generate a upgrade table when making changes to the model. https://grow.liferay.com/people/Changing+the+model+in+an+Upgrade+Process. Similar to this https://github.com/liferay-commerce/liferay-portal/pull/324